context/app/static/js/components/detailPage/related-entities/RelatedEntitiesTabs/RelatedEntitiesTabs.jsx

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 
 import { Tab } from 'js/shared-styles/tabs';
+import { entityIconMap } from 'js/shared-styles/icons/entityIconMap';
 import RelatedEntitiesTable from 'js/components/detailPage/related-entities/RelatedEntitiesTable';
-import { StyledTabs, StyledTabPanel, StyledAlert } from './style';
+import { StyledTabs, StyledTabPanel, StyledAlert, StyledSvgIcon } from './style';
 
 function RelatedEntitiesTabs({ entities, openIndex, setOpenIndex, ariaLabel, renderWarningMessage }) {
```

Comment:
I'm a little confused. Wasn't a slightly different implementation of this also in https://github.com/hubmapconsortium/portal-ui/pull/3198? Should this PR use the branch in 3198 as its base?

Reply:
It's just this file I believe, I'll be sure to carry over the change here once the other one gets approved.

Reply:
That was what I meant by the screenshot above ^